harness/tests/experiment/pytorch/test_pytorch_trial.py

```diff
@@ -393,6 +394,18 @@ def make_workloads() -> workload.Stream:
         )
         controller.run()
 
+    def test_multi_lr_schedule_one_batch(self, tmp_path: pathlib.Path) -> None:
+        example_path = "../examples/official/multiple_lr_schedulers_pytorch"
+        yaml_file = "const.yaml"
+
+        with experimental._local_execution_manager(pathlib.Path(example_path)):
+            from multiple_lr_schedulers_pytorch import MNistTrial
+
+            with open(yaml_file) as f:
+                configs = yaml.load(f, Loader=yaml.FullLoader)
```

Comment:
question: @brain-good do you have ideas on how to organize our directory so that we can directly import examples in unit tests? I think the ideal scenario is the ability to use `from examples.experimental.multiple_lr... import MNistTrial` as a top-level import, but I'm not sure that's possible without editing the circleci scripts.

Reply:
@brain-good

Reply:
I'm not really sure what about the circleci scripts would need to change. All it basically does (besides installing Python dependencies) is run `make -C harness test`.

Reply:
one idea is to test examples via something like `make -C examples test` but that can be done separately